packages/react-core/src/components/Progress/examples/ProgressHelperText.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { capitalize, Progress, ProgressProps, HelperText, HelperTextItem, Radio } from '@patternfly/react-core';
```

Comment:
Can we define a capitalize locally in the example.  The one you are importing her was really defined as an internal helper and the usage is not documented for the consumer. I have an issue open for breaking change release to update our exports to not include internal helpers.